packages/functional-tests/README.md

```diff
@@ -37,7 +37,7 @@ test('mocha tests', async ({ target, page }, info) => {
 
 ## Creating new tests
 
-`yarn record` is a convenient script to start with when creating a new test. It starts playwright in debug mode and runs the `tests/stub.spec.ts` test which creates a new account and prints the credentials to the terminal. From there you can use the Playwright inspector to record your test and copy the command output to your test file.
+`yarn record` is a convenient script to start with when creating a new test. You can use the Playwright inspector to manually perform the steps on the UI and record your test and copy the command output to your test file to get started with writing new tests.
```

Comment:
I might be confusing this slightly&hellip; this command will run <kbd>playwright test --project=stub</kbd>, but I think we deleted the stub.spec.ts file below and the "stub" project/config from the playwright.config.ts file. Not sure if this entire H2 block needs to just be deleted since I believe running <kbd>yarn record</kbd> would just throw an error.

https://github.com/mozilla/fxa/blob/9aa5bfb400cc6dcb50fb245b393da5ced3eb14fe/packages/functional-tests/package.json#L7

Reply:
Actually instead of this we should reference the `codegen` way of creating new tests as that's what I had used in my initial days. https://playwright.dev/docs/codegen#generate-tests-with-the-playwright-inspector.
Let me modify this part here in my upcoming PR. Thanks Peter for the pointer.